net-misc/ytdl-gui/ytdl_gui-2.5.recipe

```diff
@@ -51,20 +51,21 @@ BUILD_PREREQUIRES="
 	cmd:g++$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
 	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:cmake
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
-	cmd:qmake$secondaryArchSuffix >= 5
 	"
 
 BUILD()
 {
+	cmake -B build $cmakeDirArgs \
+		-DCMAKE_INSTALL_BINDIR=$appsDir
 	make build
 }
 
 INSTALL()
 {
-	mkdir -p $appsDir
-	mv ./src/youtubedl-gui $appsDir/ytdl-gui
+	make install -C build
 
```

Comment:
The files in $dataDir are useless to us (we are not linux) so best to remove them from the build:
```diff
+ 	rm -rf $dataDir
```